templates/ddd-service-orm-db/.env.example

```
@@ -1,39 +1,56 @@
 # Available options: sqlite, postgresql, mariadb, mysql, mssql, oracle
 DB_BACKEND=sqlite
 
-# SQLite (when DB_BACKEND=sqlite)
+# --- DB_BACKEND=sqlite only -------------------------------------------------------------
 DB_PATH=./data/app.db
 
-# All other backends — set DB_DSN to skip individual fields
+# --- every backend EXCEPT sqlite ---------------------------------------------------------
+# Set DB_DSN to skip the individual fields.
 DB_USER=app_user
 DB_PASSWORD=changeme
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_USER key

(UnorderedKey)